src/components/News.vue

```diff
@@ -4,7 +4,7 @@
       <div class="absolute-top-left custom-caption-top">
         <div class="subtitle text-white q-ml-sm q-mt-sm">{{item.subTitle}}</div>
         <div class="q-display-1 text-white q-ml-sm q-mb-sm">{{item.title}}</div>
-        <q-btn class=" fixed-bottom-left q-ml-sm q-mb-sm" @click="show" :label="item.label" :class="{ hidden: item.isBtnHidden }" color="white" text-color="brown" rounded>
+        <q-btn class="fixed-bottom-left q-ml-sm q-mb-sm" @click="show" :label="item.label" :class="{ hidden: item.isBtnHidden }" color="secondary" text-color="white" rounded>
```

Comment:
color="secondary" text-color="white"   -> <q-btn class="fixed-bottom-left q-ml-sm q-mb-sm bg-secondary text-neutral" ?

Reply:
only text-primary, text-secondary is available, text-neutral can not apply